XpringKit/Common/XRPLNetwork.swift

```diff
@@ -0,0 +1,6 @@
+/// Possible networks to resolve.
+public enum XRPLNetwork: String {
+  case dev = "devnet"
```

Comment:
Is it common to have lowercase enums in Swift? I know TypeScript favors uppercase enum members.

Reply:
Yes, this is recommended.

Reply:
https://docs.swift.org/swift-book/LanguageGuide/Enumerations.html